COMMIT_MSG.md

```diff
@@ -0,0 +1,92 @@
+# Git Commit/Pull-Request Message Guide
+
+## The reasons for this guide:
+
+- automatic generating of the changelog
+- simple navigation through git history (e.g. ignoring style changes)
+
+## Format of the commit message:
+```
+<type>(<scope>): <subject>
+
+<body>
+
+<footer>
+```
+
+## Example commit message:
+
+```
+fix(git): repository endpoint returns forks as well
+
+Add `fork:true` to the GitHub search api query to fix the bug.
+
+Closes #285
+```
+
+## Message subject (first line)
+
+The first line cannot be longer than 70 characters, the second line is always blank and other lines should be wrapped at 80 characters. The type and scope should always be lowercase as shown below.
+
+### Allowed <type> values:
+
+- feat (new feature for the user, not a new feature for build script)
+- fix (bug fix for the user, not a fix to a build script)
+- docs (changes to the documentation)
+- style (formatting, missing semi colons, etc; no production code change)
+- refactor (refactoring production code, eg. renaming a variable)
+- test (adding missing tests, refactoring tests; no production code change)
+- chore (updating grunt tasks etc; no production code change)
+
+### Example <scope> values:
+
+- base
+- core
+- web
+- osio-addon
+- git
+- keycloak
+- etc.
+
+The `<scope>` can be empty (e.g. if the change is a global or difficult to assign to a single component), in which case the parentheses are omitted.
+
+## Message body
+
+- uses the imperative, present tense: “change” not “changed” nor “changes”
+- includes motivation for the change and contrasts with previous behavior
+
+For more info about message body, see:
+- http://365git.tumblr.com/post/3308646748/writing-git-commit-messages
+- http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html
+
+## Message footer
```

Comment:
Missing info here?

Reply:
Ah the other items are sub-items of this.
Then I'd probably add a simple statement like "In the footer you should mention issues and any breaking changes"